cli_reference/tkn_cli/installing-tkn.adoc

```diff
@@ -13,6 +13,12 @@ You can also find the URL to the latest binaries from the {product-title} web co
 // Install tkn on Linux
 include::modules/op-installing-tkn-on-linux.adoc[leveloffset=+1]
 
+// Install tkn on Linux on IBM Z and LinuxONE
```

Comment:
With this PR merged, we would have 3 sections that are basically same, the download link is the only difference. @Preeticp would it make sense to have just one section and listing all three tarballs there?

Reply:
@ppitonak We took the same approach as we did for odo:  https://docs.openshift.com/container-platform/4.6/cli_reference/developer_cli_odo/installing-odo.html
Not sure whats better.

Reply:
@ppitonak totally makes sense. 
@SNiemann15 if you see the preview generated here: https://z_p_pipelines--ocpdocs.netlify.app/openshift-enterprise/latest/cli_reference/tkn_cli/installing-tkn.html
you will see he is right. There is no difference except the download link.
I suggest we simply retain the module: Installing Red Hat OpenShift Pipelines CLI (tkn) on Linux.
the first step could then be: 
1. Download the  relevant CLI.
    - Linux <download link>
    - Linux on IBM Z and LinuxONE <download link>
    - Linux on IBM Power Systems <download link>

I think we can do the same in odo too, but @boczkowska can take a call on that. Yana, can you please see if we can do something similar for odo?

Reply:
@ppitonak @Preeticp Seems like an additional step is needed for P and Z. Should we still try to keep it in one procedure? 

after testing both @mohit-sheth @xzheng@redhat.com said today that as step  2 `$ chmod +x tkn-linux-s390x`
is needed.

Reply:
@SNiemann15 thank you for letting us know... this step is now necessary also for x86 binaries. I'll try to find out if it would be possible to revert to old behavior